xml/http_servers.xml

```diff
@@ -1402,7 +1418,7 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:nginx:nginx:-"/>
   </fingerprint>
 
-  <fingerprint pattern="^nginx\/?(:?[\d.]+)?">
+    <fingerprint pattern="^nginx\/?(:?[\d.]+)?">
```

Comment:
As a note, I tend to run `ruby bin/recog_cleanup` before commits because it will fix pretty much all whitespace issues.